validator_client/src/attestation_service.rs

```diff
@@ -545,11 +548,15 @@ impl<T: SlotClock + 'static, E: EthSpec> AttestationService<T, E> {
                 .await
             {
                 Ok(()) => {
-                    for signed_aggregate_and_proof in signed_aggregate_and_proofs {
+                    for (signed_aggregate_and_proof, validator_index) in signed_aggregate_and_proofs
+                        .iter()
+                        .zip(validator_indices.iter())
+                    {
                         let attestation = &signed_aggregate_and_proof.message.aggregate;
                         info!(
                             log,
-                            "Successfully published attestations";
+                            "Successfully published attestation";
+                            "validator_index" => validator_index,
```

Comment:
On closer inspection I think maybe we don't need to add anything to this log, the validator index is already present as `aggregator`

Reply:
Thanks. Glad you noticed this! Done.